lib/lte_link_control/Kconfig

```diff
@@ -17,14 +17,37 @@ if LTE_LINK_CONTROL
 config HEAP_MEM_POOL_ADD_SIZE_LTE_LINK_CONTROL
 	int "Heap required for event handlers"
 	default 32
+	default 96 if LTE_LC_PDN_MODULE
 	help
 	  Extra heap size required for allocating event handlers.
 	  Default is enough for 4 handlers.
 
 # Modules: these Kconfig options enable specific features of the library.
 
+config LTE_LC_FUNCTIONAL_MODE_MODULE
+	bool "Functional mode module"
```

Comment:
API functions have these notes whenever certain API requires some modules:
`@note Requires `CONFIG_LTE_LC_PSM_MODULE` to be enabled.`

With the new Kconfig's, basically all functions need to have some note on what is required.